base/Prometheus.js

```diff
@@ -5,6 +5,7 @@
 const promClient = require('prom-client')
 const http = require('http')
 const serverUPS = require("../helpers/serverUPSHandler")
+const config = require("../config")
```

Comment:
I think config is stored under `this.client.config`

Reply:
That is not however given to Prometheus as it is extra stuff.